backend/routes/performance_routes.py

```diff
@@ -84,6 +87,21 @@ def get_adjudications(performance_id):
 def create_adjudication(performance_id):
     adjudication_json = request.get_json()
     adjudication_json['performance_id'] = performance_id
+    nomination_comments = adjudication_json['nomination_comment']
+    del adjudication_json['nomination_comment']
     new_adjudication = Adjudication.create(**adjudication_json)
```

Comment:
Can we overload this method in the `Adjudication` model and put the logic to create the `NominationComment` objects in there? This will hide the internals of our models to the "frontend" of the backend.